conda/cli/activate.py

```diff
@@ -62,7 +81,7 @@ def main():
             print(os.environ['PATH'])
             raise
         paths = []
-        sys.stderr.write("discarding %s from PATH\n" % binpath)
+        sys.stderr.write("discarding %s from PATH\n" % pathlist_to_str(binpath))
```

Comment:
After executing `conda install` with the new design for the first time, I realize that `conda.bat`, `activate.bat` and `deactivate.bat` are now placed into _every_ environment. Then it is indeed no longer necessary to always keep `<conda_root>\Scripts` in the PATH.